test/Test/Main.purs

```diff
@@ -392,6 +393,15 @@ test_kill_finalizer_bracket = assert "kill/finalizer/bracket" do
   killFiber (error "Nope") fiber
   eq "completed" <$> readRef ref
 
+test_kill_invincible ∷ Aff Unit
+test_kill_invincible = assert "kill/invincible" do
+  true <$ do
+    f <- forkAff $
+      invincible $
+        delay $ Milliseconds 50.0
+    delay $ Milliseconds 10.0
+    killFiber (error "crash") f
```

Comment:
I'm not sure exactly what this is testing. It does not verify behavior in any way.